adapters/nft-key/index.js

```diff
@@ -0,0 +1,164 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const Ethereum = require("../../sdk/EVMC");
```

Comment:
Use `avalanche.js` SDK for Avalanche protocol.

Reply:
I see - got it.